File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/generatedComments/morechildren/Json.java
package io.github.gusandrianos.foxforreddit.data.models.generatedComments.morechildren;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Json{

    @SerializedName("data")
    private DataThings data;

    @SerializedName("errors")
    private List<Object> errors;

    public DataThings getData(){
        return data;
    }

    public List<Object> getErrors(){
        return errors;
    }
}

File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/singlepost/morechildren/Json.kt
package io.github.gusandrianos.foxforreddit.data.models.singlepost.morechildren

import com.google.gson.annotations.SerializedName

class Json {
    @SerializedName("data")
    val data: DataThings? = null

    @SerializedName("errors")
    val errors: List<Any>? = null
}